Taggers/python/flashggTagSorter_cfi.py

```diff
@@ -20,7 +20,7 @@
 #        cms.PSet(TagName = cms.InputTag('flashggVHLooseTag')),
         cms.PSet(TagName = cms.InputTag('flashggZHLeptonicTag')),
         cms.PSet(TagName = cms.InputTag('flashggWHLeptonicTag')),
-        cms.PSet(TagName = cms.InputTag('flashggVHLeptonicLooseTag')),
+#        cms.PSet(TagName = cms.InputTag('flashggVHLeptonicLooseTag')),
```

Comment:
please cleaup up everything related to VH that is no longer in use (comments will create confusion in the future)

Reply:
OK. I have removed VH tags which are not used.